cloudfoundry.md

```diff
@@ -130,6 +130,8 @@ cf set-env foia DATABASE_URL [value]
 cf set-env foia FOIA_ANALYTICS_ID [value]
 cf set-env foia FOIA_SECRET_SESSION_KEY [value]
 cf set-env foia DJANGO_SETTINGS_MODULE foia_hub.settings.dev
+cf set-env foia NEW_RELIC_LICENSE_KEY=[value]
+cf set-env foia NEW_RELIC_APP_NAME=[value]
```

Comment:
Now that we've merged in the zero-downtime deploy PR, `foia` can be removed in favor of `foia-a` and `foia-b`. This also means that foia.cf.18f.us stops getting updated, and we don't have a stable URL for the CF-hosted app until we get our production domain. All the more reason to have a stable staging CF-hosted app, where multi-minute downtime deploys are acceptable.